### PR TITLE
Replace language python script

### LINKED ISSUE #17: Convert buid translation script to nodejs
Right now we have one script that generates the dictionaries, and this script is written in Python. To avoid that need that our collaborators must have Python installed in order to be able to run this script it is better if we convert the script to Javascript, to be executed by NodeJS.

This is the Python Script:
https://github.com/sumelms/frontend/blob/main/utils/build_translations.py

### PR DESCRIPTION
**Pull request by rluders**
Replace the Python script used to generate the translation with Javascript script.

Closes #17

### CODE REVIEW

**Comment by rluders** [comment]
@arianerocha see? The LGTM is mad 'cause I removed the Python script. LoL

File: utils/translations.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    };
+  });
+};
+
+const generateFiles = () => {
+  Object.keys(languages).map((lang) => {
+    const langPath = path.join(LANG_DIR, lang);
+
+    if (!fs.existsSync(langPath)) {
+      fs.mkdirSync(langPath);
```
Comment (arianerocha) [inline utils/translations.js:51]: ` fs.mkdirSync(langPath, { recursive: true});`

File: utils/translations.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    if (languages[lang] === undefined) {
+      const obj = {};
+      obj[module] = {};
+      obj[module][file] = {};
+      languages[lang] = obj;
+    }
+
+    if (languages[lang][module] == undefined) {
```
Comment (arianerocha) [inline utils/translations.js:32]: `if (languages[lang][module] === undefined) {`

File: utils/translations.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+let languages = {};
+
+const searchLanguageFiles = () => {
+  glob.sync(`${SRC_DIR}/**/translations/**/*.json`).map((globFilePath) => {
+    const paths = globFilePath.split("/").reverse();
+
+    const file = paths[0].toLowerCase().replace(".json", "");
+    const lang = paths[1].toLowerCase();
+    const module = paths[3] == "src" ? "__DEFAULT__" : paths[3].toLowerCase();
```
Comment (arianerocha) [inline utils/translations.js:16]: ` paths[3] === "src"`

File: utils/translations.js (hunk 2 above)
Comment (rluders) [reply utils/translations.js:32]: Ops

File: utils/translations.js (hunk 3 above)
Comment (rluders) [reply utils/translations.js:16]: Ops, I did it again.

File: utils/translations.js (hunk 1 above)
Comment (rluders) [reply utils/translations.js:51]: Well, it doesn't make that difference 'cause `lang`path will always exists, but... also doesn`t create any negative impact. So, OK, let's do it. :+1:

**Pull request merged**